docs/README.md

```diff
@@ -1,80 +0,0 @@
-FortiSOAR Employee On/Off Boarding solution pack helps automating this process by provisioning the access for new employees on the systems they have access to and disabling that access for the ones leaving the organization.
-The mapping of each user access scope is done via Active Directory, where the group object holds the list of resources the members of the group can access.
-Typically user provisioning order comes from HR, in this simulation we assume it comes via Email. The required attributes for new users are:
-
-- Username
-- Group
-- Title
-
-FortiSOAR will: 
- - Create the user on *Active Directory*
- - Create a mailbox for the user on *Microsoft Exchange*
- - Read the group access scope from *Active Directory* and provision access for the user on all systems part of the above scope
-
- > The group/resources mapping is managed via the *Info* field of the group definition within *Active Directory*. the field contains the CSV of UUIDs of the management playbooks for the resources within the scope of the group. As an example, if the group grants access to SSH/Server1 and MySQL/Server2 the info field would look like: `9c827fcf-53bd-403a-b3bf-406f1a27d9d8,0d11f906-15ea-4aa9-a665-405cd44f061f` 
-
-If the action is to **deprovision** similar logic is applied except the action would be either to disable the user or delete it depending on the target system
-
-## 2) Prerequisites
-
-### 2.1) Generic Prerequisites:
-- SOAR Essentials Solution Pack
-
-### 2.2) For Simulation mode (Fake mock data):
-- Make sure your global variable: `Demo_mode` is set to `true` (Playbook editors > Tools > Global Variables)
-- Make sure your global variable: `EOF_SYS_User` is set to a username of your choice (Playbook editors > Tools > Global Variables). The variable holds the name of the newly provisioned or deprovisioned user
-- Make sure your global variable: `EOF_DB_User` is set to a username of your choice (Playbook editors > Tools > Global Variables)
-- FortiSOAR SOC Simulator Connector configured (1.0.9+)
-- It is recommended to have a working SMTP connector to receive tasks notifications
-
-### 2.3) For Live mode (Real data):
-- MS Exchange/SMTP Connectors configured with your MS Exchange parameters (To receive instruction emails and send notifications)
-- AD Connector configured with your DC parameters (To run AD related actions)
-- MySQL/SSH server(s) to act as resources
-
-> All Credentials used with the above connectors must have the required privileges to perform the actions they are meant to do (add user, disable user...etc)
-
-> You can customize the use case for your environment by adding 3 playbooks for each service: Provision, Deprovision , Management the latter is only used as a wrapper for the first two. Open any examples of the existing services to see how is it done
-
-
-### 3) Installation/Deployment:
```

Comment:
install process is not correct .

Reply:
Docs for this SP is getting changed, will pull once changes are made